tools/tdbg/commands.go

```diff
@@ -472,3 +473,32 @@ func AdminRefreshWorkflowTasks(c *cli.Context) error {
 	}
 	return nil
 }
+
+// AdminRebuildMutableState rebuild a workflow mutable state using persisted history events
+func AdminRebuildMutableState(c *cli.Context) error {
+	adminClient := cFactory.AdminClient(c)
+
+	namespace, err := getRequiredGlobalOption(c, FlagNamespace)
```

Comment:
Please rebase on master branch. I believe I renamed this function (as cli/v2 doesn't have separate function for getting global flag values).